bioimageio/core/prediction_pipeline/statistical_measures.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass
+from typing import Any, Optional
+
+
+@dataclass(frozen=True)
+class Measure:
+    # value: Any  # not specified here to keep clean arg order, e.g. Percentile(50) == Percentile(n=50)
+    def __post_init__(self):
+        assert hasattr(self, "value")  # monkey patch for abstract dataclass
+
+
+@dataclass(frozen=True)
+class Mean(Measure):
+    value: Optional[float] = None
+
+
+@dataclass(frozen=True)
+class Std(Measure):
+    value: Optional[float] = None
+
+
+@dataclass(frozen=True)
+class Percentile(Measure):
+    n: float
+    value: Optional[float] = None
```

Comment:
what would be the meaning of `value` here?

Reply:
the computed percentile